backend/src/api/account/account.service.js

```diff
@@ -14,30 +14,43 @@ import { randomUUID } from 'crypto';
 import { CacheService } from '../../services/index.js';
 import { ACCOUNT_STATE } from '../../shared/enum/index.js';
 
-const { User, Account, Role } = sequelize;
+const { User, Account, Role, StudentClass } = sequelize;
 
 class AccountService {
     async getUserOfAccount(accountId) {
-        const user = await User.findAll({
-            where: {
-                userId: accountId
-            }
-        });
-        return user.map((x) => AccountDto.toDto(x));
+        try {
+          const user = await User.findOne({
+            where: { userId: accountId }
+          });
+          if (!user) {
+            throw new Error("User not found");
+          }
+          return AccountDto.toDto(user);
+        } catch (error) {
+          console.error(error);
+          throw error;
+        }
+      }
+
+    async getAllAccounts() {
+        const data = await Account.findAll();
+        return data.map((x) => AccountDto.toDto(x));
     }
 
-    async getAllAccountsOfRole(roleId) {
-        const role = await Role.findByPk(roleId);
-        if (role == null) {
-            throw new EntityNotFoundException('Role', roleId);
+    async getAllStudentsByClassId(classId) {
+        const studentClassEntity = await StudentClass.findByPk(classId);
```

Comment:
find all record that has classId equal to classId not single one, classId is not primary key of StudentClass Entity, it's just FK :(

Reply:
hint: find all record in StudentClass that has classId equal to classId passed to function using `where`, load all Student related to this entity through `include` in query options.